web/.env.example

```diff
@@ -1,28 +1,27 @@
-APP_NAME=Laravel
+APP_NAME=Wikijump
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
-APP_URL=http://localhost
+APP_URL=https://www.wikijump.dev/wikijump
+ASSET_URL=https://www.wikijump.dev/wikijump/assets
```

Comment:
Noting this prohibits the effective creation of a page with slug `wikijump`. We will need to create a psuedo-page in that location similar to what will be needed for `_admin`.

Reply:
This is not intended to be a long-term solution in all honesty, but I haven't decided on which way to take it. But this way does at least allow for separation of the two codebases for the time being.

Reply:
I mostly bring it up since generally special routes get some kind of `--` in it (e.g. `files--common`, `api--v1`). This is fine for now though.

Reply:
If we wish to move it around later, only this file and the `install/*docker/php-fpm/etc/nginx/sites-available/wikijump` need to be changed, and I can't see any reason why adding `--` to the path would be problematic, but I'm also not sure if I want to standardize on that as the design pattern we want to use.

Reply:
I'm definitely open to alternatives to `--` being used, I've just used it for now since it's what Wikidot uses, and honestly it's a relatively elegant solution to avoid path collision with arbitrary page names.

Reply:
> 
> 
> Noting this prohibits the effective creation of a page with slug `wikijump`. We will need to create a psuedo-page in that location similar to what will be needed for `_admin`.

Also just to clarify, even if a page was created at `wikijump`, nginx is sending it to Laravel before the Wikidot code is even aware there's a request. So you'd have essentially an unreachable page for now.

Reply:
Right, but this has some side effects such as any local links pointing to the page appearing as orange, even though there is a valid path there. This phenomenon is documented in [WJ-331](https://scuttle.atlassian.net/browse/WJ-331).